bundles/framework/announcements/view/AnnouncementsCollapse.jsx

```diff
@@ -18,29 +19,33 @@ const AnnouncementsCollapse = ({controller, checked, announcements, modals }) =>
               <AnnouncementsModal
               id={modal.id}
               controller={controller}
-              title={modal.title}
-              content={modal.content}
+              title={modal.locale[lang].name}
+              content={modal.locale[lang].content}
```

Comment:
Would be nice to use Oskari.getLocalized() here as well, but we'll change it later.

Reply:
Just now learned about it so I think I switched everything I found to that on the editor PR